lib/faker/date.ex

```diff
@@ -62,11 +62,6 @@ defmodule Faker.Date do
 
   @doc """
   Returns a random date in the past up to N days, today not included
-
-  ## Examples
-
-      iex> Faker.Date.backward(4)
-      #=> ~D[2016-12-20]
```

Comment:
I am not sure if we should have doctests for `backward` and `forward`, because I think these doctests will break when the day is over and we'll need to update the output again :/

Reply:
Yeah, we need to think about it